packages/fx-core/src/plugins/resource/bot/errors.ts

```diff
@@ -68,7 +106,7 @@ export class PluginError extends Error {
 
 export class PreconditionError extends PluginError {
   constructor(message: string, suggestions: string[]) {
-    super(ErrorType.User, ErrorNames.PRECONDITION_ERROR, message, suggestions);
+    super("User", ErrorNames.PRECONDITION_ERROR, message, suggestions);
```

Comment:
If there are many place use "User" or "System", we could make them as constant if you don't like string-based enum. This would help us easy to update the string value in one place if there is any change.